api/EXT_sRGB.json

```diff
@@ -10,7 +10,7 @@
           },
           "chrome_android": "mirror",
           "edge": {
-            "version_added": false
+            "version_added": "80"
```

Comment:
Not 79?

Reply:
Edge had disabled a few WebGL extensions in its first release.  This is one of those extensions that was disabled.

Reply:
Ah right, that sounds familiar.